sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -107,3 +109,28 @@ def test_check_estimators_unfitted():
     # check that CorrectNotFittedError inherit from either ValueError
     # or AttributeError
     check_estimators_unfitted("estimator", CorrectNotFittedErrorClassifier)
+
+
+def test_common():
+    # check that the fit function of an estimator will accept
+    # a 'sample_weight' parameter of type pandas.Series
+    try:
+        import pandas as pd
+    except ImportError:
+        raise SkipTest("Pandas not found")
+    X = pd.DataFrame([[1, 1], [1, 2], [1, 3], [2, 1], [2, 2], [2, 3]])
+    y = pd.Series([1, 1, 1, 2, 2, 2])
+    weights = pd.Series([1] * 6)
+    for estimator_name, Estimator in all_estimators():
+        estimator = Estimator()
+        if has_fit_parameter(estimator, "sample_weight"):
+            try:
+                # default solver liblinear doesn't support this parameter
```

Comment:
solver='liblinear' does support `sample_weight` . Where did you get this idea from?